### PR TITLE
Data/message embeed

### PR DESCRIPTION
**Pull request by DoviMaj**
- [x] You have read the [Odin Bot README/Contributing Guide](https://github.com/TheOdinProject/odin-bot-v2/blob/master/README.md).

#### 1.Describe the changes made:

Update data command to MessageEmbeed. add `nodemon` dependency and `dev` script.

data-3

#### 2. If this PR modifies a command, please provide a screenshot of the passing tests below. NOTE: Your PR will not be merged if you have any failing test cases. 

tests

#### 3. If this PR is related to an open issue please reference it with the `#` operator and the issue number below:

https://github.com/TheOdinProject/odin-bot-v2/issues/145

### CODE REVIEW

File: botCommands/simpleCommands.js [hunk 1]
```diff
@@ -43,7 +43,31 @@ registerBotCommand(question.regex, question.cb);
 
 const data = {
   regex: /(?<!\S)\/data(?!\S)/,
-  cb: () => '**Please state your question in the form of a question! https://www.dontasktoask.com/**',
+  cb: () => {
+    const dataEmbeed = new Discord.MessageEmbed()
```
Comment (twalton83) [inline botCommands/simpleCommands.js:47]: I'd love if you fixed the typo "dataEmbeed", should be "dataEmbed".

**Review by twalton83 — CHANGES_REQUESTED** [review]
One small typo and we're good to go.

File: botCommands/simpleCommands.js (hunk 1 above)
Comment (DoviMaj) [reply botCommands/simpleCommands.js]: Done

**Review by twalton83 — APPROVED** [review]
Thank you.

**Pull request merged**